README.md

```diff
@@ -1,87 +1,129 @@
 
-For example:
+The incident_mangement class also lets you specify additional (but optional) incident fields like the `category`, `subcategory` and `assigned_to` fields via the corresponding `category`, `subcategory`, and `assigned_to` parameters, respectively. See the `REFERENCE.md` for the full details.
 
-```
-### `pet::cat`
+Each incident will include the following information provided by Puppet:
+* __Caller__: The user specified by the sys_id given to the `caller_id` parameter of the incident_management class. This can be any user in Servicenow and doesn’t need to be the same as the user that creates the incident via the username/password or oauth_token parameters.
+* __Urgency__: Defaults to 3 - Low. Configurable
+* __Impact__: Defaults to 3 - Low. Configurable
+* __State__: Defaults to New. Configurable
+* __Short Description__: Contains the following information
+  * __Status__: changed, unchanged, failed, pending changes
+  * __Node__: the fqdn of the node the agent ran on
+  * __Report Time__: The timestamp of the report to help find the report in the console
+* __Description__: See the description section from event management above. Incidents will get the same description
 
-#### Parameters
+## Troubleshooting
 
-##### `meow`
+To verify that everything worked, trigger a Puppet run on one of the nodes in the `PE Master` node group then log into the node. Once logged in, run `sudo tail -n 60 /var/log/puppetlabs/puppetserver/puppetserver.log | grep servicenow`. You should see some output. If not, then the class is probably not being classified properly. Either the class is not being assigned to the Puppet server nodes at all, or there may be catalog compilation errors on those nodes with the provided parameter values. Please use GitHub issues to file any bugs you find during your troubleshooting.
 
-Enables vocalization in your cat. Valid options: 'string'.
+## Development
 
-Default: 'medium-loud'.
+### Unit tests
+To run the unit tests:
+
+```
+bundle install
+bundle exec rake spec/unit
+bundle exec rake spec/classes
```

Comment:
This should probably just be one call to `bundle exec rake spec_prep spec`

Supplying paths like this assumes a call to `bundle exec rspec` not rake. And calling to `rake spec` will run all tests in `spec/unit` and `spec/classes`, while the only way to avoid two calls when calling to rspec would  be to run first run `bundle exec rake spec_prep` and then run `bundle exec rspec spec/unit spec/classes`